ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/vxlan_interface.py

```diff
@@ -93,7 +94,7 @@ def vxlan_interface(self) -> dict | None:
                                 tenant, "evpn_l3_multicast.evpn_underlay_l3_multicast_group_ipv4_pool_offset", default=0
                             )
                             offset = vni - 1 + underlay_l3_mcast_group_ipv4_pool_offset
-                            vrfs[vrf_name]["multicast_group"] = self._avd_ip_addressing._ip(underlay_l3_multicast_group_ipv4_pool, 32, offset, 0)
+                            vrfs[vrf_name]["multicast_group"] = get_ip_from_pool(underlay_l3_multicast_group_ipv4_pool, 32, offset, 0)
```

Comment:
so should we not have some kind of way to access get_ip_from_pool through the `self._avd_ip_addressing._ip` still? @ClausHolbechArista

Reply:
@gmuloc I think the better solution here would be to create a method for `underlay_l3_multicast_group_ipv4` in `avd_ip_addressing`.

Reply:
Agreed - @Vibhu-gslab could you please create a method for this in avd_ip_addressing and use it here? and same for l2multicast_group below?